bundles/org.openhab.binding.webexteams/src/main/java/org/openhab/binding/webexteams/internal/api/AccessTokenRequest.java

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2010-2022 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.webexteams.internal.api;
+
+import java.net.URI;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+
+import com.google.gson.annotations.SerializedName;
+
+/**
+ * @author Tom Deckers - Initial contribution
+ */
+@NonNullByDefault
+class AccessTokenRequest {
+    @SerializedName("grant_type")
```

Comment:
These SerializedName are not necessary if you use the GsonBuilder with a FieldNamingPolicy.

Reply:
The Webex API uses two different types of 'field naming policy'.  For the Token exchange it uses underscores, for the regular API it uses camel casing.  I use the `doRequest` method in `WebexTeamsApi.java` for both.  So I'd have to add additional logic in that method to differentiate FieldNamingPolicy based on which class is being translated.  So I though maybe easier to understand to keep the `@SerializeName` in the class definitions.
Would that work, or do you have any other suggestion/preference to make this cleaner?

Reply:
I've reworked the oauth flow to use Openhab OAuthFactory and the files that contained these annotations have now been removed.